tests/modules/submodules/test_oracle_module.py

```diff
@@ -16,13 +16,6 @@ def execute_module(self, blockstamp):
         return ModuleExecuteDelay.NEXT_FINALIZED_EPOCH
 
 
-@pytest.fixture(autouse=True)
```

Comment:
No need to real sleep in tests
Probably we should mock this var from `variables`